stories/TextInput.jsx

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
why remove?

Reply:
Because I could not figure out the issue.

Other places that used with counter worked fine (and I think that might be because the storybook file was not flowified, or it could be something else)